test/Tests/src/Data/Time/Date_Part_Spec.enso

```diff
@@ -0,0 +1,99 @@
+from Standard.Base import all
+
+import Standard.Test
+
+spec name create_new_date =
+    Test.group (name + "date part tests")  <|
+        Test.specify "should return if a leap year" <|
+            create_new_date 2022 8 25 . is_leap_year . should_equal False
+            create_new_date 1999 12 31 . is_leap_year . should_equal False
+            create_new_date 1996 6 19 . is_leap_year . should_equal True
+            create_new_date 2000 1 1 . is_leap_year . should_equal True
+            create_new_date 1900 1 1 . is_leap_year . should_equal False
```

Comment:
I'd consider adding tests for `1582`, a few years prior to 1500 and maybe also BC? Although I do appreciate we don't want to drown in test cases either.